CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix `values.yaml` to set NTP settings in `.network.ntp` and adapt template.
```

Comment:
This isn't a fix. This is a breaking change and that should be boldly noted in the changelog.

Reply:
ACK, I'll provide a PR to improve the changelog.

Reply:
FYI I saw it as a fix because the change is an alignment between values.yaml and values.schema.json, which were out of sync.

Reply:
Here: https://github.com/giantswarm/cluster-cloud-director/pull/134